server/models/index.js

```diff
@@ -1,8 +1,13 @@
-const db = require('../db/db.js');
+const { pool } = require('../db/db.js');
 
 module.exports = {
-  getRelated: () => {
-
+  getRelated: async () => {
+    try {
+      const res = await pool.query("SELECT * FROM related");
+      console.log(res.rows);
```

Comment:
Good work getting things connected. I'm assuming to be able to query related there was some table creation done through postgres' CLI?